src/newViews/ProjectEdit/GeoAreas/RegionMapList/index.tsx

```diff
@@ -42,17 +105,20 @@ function RegionMapList(props: Props) {
         id,
     }), []);
 
-    if (regions.length < 1) {
-        return (
-            <div className={_cs(styles.message, className)}>
-                <IoMapOutline className={styles.icon} />
-                {_ts('geoAreas', 'noGeoAreas')}
-            </div>
-        );
-    }
-
     return (
         <div className={_cs(styles.regionMapList, className)}>
+            <RegionSelectInput
+                className={_cs(styles.input, styles.region)}
+                name="regions"
+                activeProject={activeProject}
+                value={selectedRegion}
```

Comment:
I don't know if we need to pass value here

Reply:
@AdityaKhatri what do you think?

Reply:
I think you're right, we don't really need the value.